[2024-12-25T18:32:41.760+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-25T18:32:41.796+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: realtime_dump_pipeline.realtime_pipeline manual__2024-12-25T18:32:36.915962+00:00 [queued]>
[2024-12-25T18:32:41.815+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: realtime_dump_pipeline.realtime_pipeline manual__2024-12-25T18:32:36.915962+00:00 [queued]>
[2024-12-25T18:32:41.818+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-25T18:32:41.850+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): realtime_pipeline> on 2024-12-25 18:32:36.915962+00:00
[2024-12-25T18:32:41.869+0000] {standard_task_runner.py:72} INFO - Started process 3482 to run task
[2024-12-25T18:32:41.880+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'realtime_dump_pipeline', 'realtime_pipeline', 'manual__2024-12-25T18:32:36.915962+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/data_dump_realtime.py', '--cfg-path', '/tmp/tmpm11pilfh']
[2024-12-25T18:32:41.886+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask realtime_pipeline
[2024-12-25T18:32:42.007+0000] {task_command.py:467} INFO - Running <TaskInstance: realtime_dump_pipeline.realtime_pipeline manual__2024-12-25T18:32:36.915962+00:00 [running]> on host 8e07e13977c9
[2024-12-25T18:32:42.211+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='haziq' AIRFLOW_CTX_DAG_ID='realtime_dump_pipeline' AIRFLOW_CTX_TASK_ID='realtime_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T18:32:36.915962+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-25T18:32:36.915962+00:00'
[2024-12-25T18:32:42.217+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-25T18:32:42.221+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-25T18:32:42.225+0000] {logging_mixin.py:190} INFO - Current task name:realtime_pipeline state:running start_date:2024-12-25 18:32:41.798870+00:00
[2024-12-25T18:32:42.230+0000] {logging_mixin.py:190} INFO - Dag name:realtime_dump_pipeline and current dag run status:running
[2024-12-25T18:32:42.234+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-25T18:32:51.315+0000] {logging_mixin.py:190} INFO - <pyspark.sql.session.SparkSession object at 0x7fe98dedc040>
[2024-12-25T18:32:51.317+0000] {logging_mixin.py:190} INFO - Spark session created successfully
[2024-12-25T18:32:54.499+0000] {logging_mixin.py:190} INFO - Reading from topic stock
[2024-12-25T18:32:54.981+0000] {java_gateway.py:2273} INFO - Callback Server Starting
[2024-12-25T18:32:54.985+0000] {java_gateway.py:2275} INFO - Socket listening on ('127.0.0.1', 43771)
[2024-12-25T18:32:58.836+0000] {clientserver.py:561} INFO - Python Server ready to receive messages
[2024-12-25T18:32:58.838+0000] {clientserver.py:575} INFO - Received command c on object id p0
[2024-12-25T18:33:01.493+0000] {warnings.py:109} WARNING - /opt/***/etl/data_dump_v2.py:88: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  pandas_df.to_sql(

[2024-12-25T18:33:01.497+0000] {logging_mixin.py:190} INFO - Error writing batch 0: 'Engine' object has no attribute 'cursor'
[2024-12-25T18:33:42.911+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-12-25T18:33:42.914+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-25T18:33:42.919+0000] {process_utils.py:132} INFO - Sending 15 to group 3482. PIDs of all processes in the group: [3484, 3482]
[2024-12-25T18:33:42.921+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 3482
[2024-12-25T18:33:42.924+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-25T18:33:42.972+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-12-25T18:33:42.976+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-25T18:33:43.425+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=3482, status='terminated', exitcode=0, started='18:32:41') (3482) terminated with exit code 0
[2024-12-25T18:33:43.523+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=3484, status='terminated', started='18:32:41') (3484) terminated with exit code None
